src/transforms/code-scanning-alert.ts

```diff
@@ -0,0 +1,100 @@
+import issueKeyParser from "jira-issue-key-parser";
+import {Context} from "probot/lib/context";
+import {JiraRemoteLinkData, JiraRemoteLinkStatusAppearance} from "../interfaces/jira";
+import logger from "../config/logger";
+
+const MAX_STRING_LENGTH = 255;
+
+const getPullRequestTitle = (repoName: string, prId: number, repoOwner: string, context: Context): Promise<string> => {
+	return context.github.pulls.get({
+		owner: repoOwner,
+		repo: repoName,
+		pull_number: prId
+	}).then((response) => {
```

Comment:
According to our codestyle conventions we favour "await" rather than "then". Should change it to await and try/catch?

Reply:
Did you miss this change? I'm still seeing .then as opposed to await.